README.md

```diff
@@ -1,36 +1,28 @@
+# gateway-api-integrator-operator
 [![CharmHub Badge](https://charmhub.io/gateway-api-integrator/badge.svg)](https://charmhub.io/gateway-api-integrator)
 [![Publish to edge](https://github.com/canonical/gateway-api-integrator-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/gateway-api-integrator-operator/actions/workflows/publish_charm.yaml)
 [![Promote charm](https://github.com/canonical/gateway-api-integrator-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/gateway-api-integrator-operator/actions/workflows/promote_charm.yaml)
 [![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 
-A Juju charm that deploys and manages a [Gateway API integator](https://gateway-api.sigs.k8s.io/) on Kubernetes.
-Gateway API is an official Kubernetes project focused on L4 and L7 routing in Kubernetes. This
-project represents the next generation of Kubernetes Ingress, Load Balancing,
-and Service Mesh APIs. From the outset, it has been designed to be generic,
-expressive, and role-oriented.
+A Juju charm that deploys and manages a [Gateway API integator](https://gateway-api.sigs.k8s.io/) on Kubernetes. Gateway API is an official Kubernetes project focused on L4 and L7 routing in Kubernetes. This project represents the next generation of Kubernetes Ingress, Load Balancing, and Service Mesh APIs. From the outset, it has been designed to be generic, expressive, and role-oriented. This charm simplifies initial deployment and "day N" operations of Gateway API Integrator. It allows for deployment on many different Kubernetes platforms, including [Canonical K8s](https://ubuntu.com/kubernetes). 
```

Comment:
I'd prefer to remove jargon like "day N" whenever possible, so possibly rephrase the beginning of that sentence to read "This charm simplifies deployment and operations..."

Reply:
thanks, I updated this.